### ISSUE TITLE
We're not validating a proper email is given in invitation form

### ISSUE THREAD

**Issue by julian-berbel** [mumuki/mumuki-laboratory#1327]

**Comment by julian-berbel** [comment]
rethinking this though, what about users with no email?

**Comment by flbulgarelli** [comment]
> rethinking this though, what about users with no email?

I would still make it mandatory

**Comment by julian-berbel** [comment]
email became readonly in invitation forms here https://github.com/mumuki/mumuki-laboratory/commit/380957651d77c1855ea0b037d6aaeb7c290c16bb.
So maybe this issue is not a problem anymore?
I think this is enforced only on client though. We should add that logic somewhere in server too.

**Comment by luchotc** [comment]
I agree! We should validate the presence of an email server-side too
